{{cookiecutter.project_name}}/requirements_test.txt

```
@@ -0,0 +1,3 @@
+-r requirements_dev.txt
+pytest
+pytest-homeassistant-custom-component
```

With a fresh install I have the following message when installing the requirements as described in `CONTRIBUTING.md`:
``` console
pytest-homeassistant-custom-component 0.1.0 requires pytest==6.1.2, but you'll have pytest 6.2.1 which is incompatible.
```
You should pin the version of the requirements.
From what I understand `pytest-homeassistant-custom-component` requires `homeassistant` and `pytest`.  So theoretically no need to add them here. But as homeassistant is not pinned in `pytest-homeassistant-custom-component` installing it before avoid any version conflict.
```suggestion
-r requirements_dev.txt
pytest-homeassistant-custom-component==0.1.0
```